src/Select.js

```diff
@@ -339,31 +340,38 @@ const Select = React.createClass({
 			});
 		}
 
+		const inputLabel = this.getInputLabel();
+
+		// determines if the user mouse downed on the actual ValueComponent excluding placeholder text
+		const isOnValue = this.valueComponent && (this.valueComponent.valueNode === event.target) ? true : false;
```

Comment:
This probably could be simplified to just be `this.valueComponent && (this.valueComponent.valueNode === event.target)`